### PR TITLE
DDR Commands Edit

### PR DESCRIPTION
**Pull request by Ailloviee**
Editted command descriptions for `!findallreads` and `!dumpmodulereads`
commands

Fixed some documentation errors

Signed-off-by: Charles_Zheng <Juntian.Zheng@ibm.com>

### CODE REVIEW

**Comment by tajila** [comment]
@keithc-ca can you please review this

File: debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/DumpModuleReadsCommand.java [hunk 1]
```diff
@@ -40,21 +40,21 @@
...
+ * DumpModuleReads command displays all modules that require the target module
  * 
  * Example:
  * !dumpmodulereads 0x00000130550C7E88
  * Example output: 
  * !j9module 0x000001305F46DB38
  * !j9module 0x00000130550DBB58
  * !j9module 0x000001305F469EA8
- * .....(A list of every module read by the target module)
+ * .....(A list of every module that requires the target module)
```
Comment (keithc-ca) [inline debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/DumpModuleReadsCommand.java]: If this command is about querying which modules are 'required' by a particular module, then the class and the name of the command it implements should also be adjusted. Likewise for `FindAllReadsCommand`.

Comment (Ailloviee) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/DumpModuleReadsCommand.java]: Do you want me to maybe change their names to  `dumpmodulerequires` and `dumprequiresmodule`?

Comment (pshipton) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/DumpModuleReadsCommand.java]: @keithc-ca ?

Comment (keithc-ca) [reply debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/DumpModuleReadsCommand.java]: I spoke with @Ailloviee and @tajila suggesting that we should have an issue that captures a plan for the set of new module-related DDR commands so we can be avoid changing command names (more than once). I haven't seen that issue yet and so it's unclear (to me) what the final names should be.

**Comment by keithc-ca** [comment]
No recent activity - closing.

**Pull request closed without merging**